taca/storage/storage.py

```diff
@@ -114,3 +218,31 @@ def _send_to_swestore(f, dest, remove=True):
         else:
             LOG.info("Run {} is not {} days old yet. Not archiving".format(run, str(days)))
     os.remove("{}.archiving".format(run.split('.')[0]))
+
+
+def get_closed_projects(projs,pj_con,days,logger=None):
```

Comment:
no that wouldn't work, you would have to execute `LOG = get_logger()` in `do_another_thing()` too, this is called "scope of variables" there are lots of docs about this out there if you're bored ;-)